core/commands/show_command.rb

```diff
@@ -0,0 +1,258 @@
+# frozen_string_literal: true
+
+require_relative '../models/network_settings'
+require_relative '../models/configuration'
+
+# Command shows information for the user.
+class ShowCommand < BaseCommand
+
+  # List of actions that are provided by the show command.
+  SHOW_COMMAND_ACTIONS = {
+    box: {
+      description: 'Show box name based on the path to the configuration file',
+      action: ->(*params) { show_box_name_in_configuration(*params) }
+    },
+    boxes: {
+      description: 'List available boxes',
+      action: ->(*) { show_boxes }
+    },
+    boxinfo: {
+      description: 'Show the field value of the box configuration',
+      action: ->(*) { show_box_field }
+    },
+    boxkeys: {
+      description: 'Show keys for all configured boxes',
+      action: ->(*) { show_box_keys }
+    },
+    keyfile: {
+      description: 'Show box key file to access it',
+      action: ->(*params) do
```

Comment:
Extract lambda blocks into separate methods. This hash should just redirect the call to the specified method.